FlickrViewer/FlickrViewer/PhotoViewController.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  PhotoViewController.swift
+//  FlickrViewer
+//
+//  Created by Kirill Shteffen on 06/07/2018.
+//  Copyright © 2018 BlackBricks. All rights reserved.
+//
+
+import UIKit
+import Foundation
+
+class PhotoViewController: UIViewController,UITableViewDataSource, UITableViewDelegate {
+    
+    @IBOutlet weak var photoImageView: UIImageView!
+    @IBOutlet weak var tableView: UITableView!
+    
+    
+    var commentaries: [Commentary] = []
+    var flickrPhoto: FlickrPhoto?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.dataSource = self
+        tableView.delegate = self
+        if flickrPhoto != nil {
+            photoImageView.sd_setImage(with: flickrPhoto!.photoUrl as URL?)
```

Comment:
flickrPhoto! force unwrap can be reason for random crashes. Need proper handling by using "guard"